pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -266,8 +266,8 @@ func getMinRowNum(runs []models.Run) models.RowNum {
 
 // renumberRows will update the runs.row_num field with the correct ordinal
 func (r RunRepository) renumberRows(tx *gorm.DB, startWith models.RowNum) error {
-	if startWith <= models.RowNum(0) {
-		return eris.Errorf("attempting to renumber with 0 or less row number value")
+	if startWith < models.RowNum(0) {
+		return eris.Errorf("attempting to renumber with less than 0 row number value")
```

Comment:
nice simple fix :-) Can you add a test for this situation? I think `tests/integration/golang/aim/run/delete_test.go` is verifying the renumbering, but it only exercises delete when there are a few records, not a single record.